src/core/src/bootstrap/Constants.py

```diff
@@ -334,4 +336,11 @@ class UbuntuProClientSettings(EnumBackport):
         MINIMUM_PYTHON_VERSION_REQUIRED = (3, 5)  # using tuple as we can compare this with sys.version_info. The comparison will happen in the same order. Major version checked first. Followed by Minor version.
         MAX_OS_MAJOR_VERSION_SUPPORTED = 18
         MINIMUM_CLIENT_VERSION = "27.14.4"
+    class StatusTruncationConfig(EnumBackport):
+        INTERNAL_FILE_SIZE_LIMIT_IN_BYTES = 126 * 1024
+        AGENT_STATUS_FILE_SIZE_LIMIT_IN_BYTES = 128 * 1024
+        MIN_TRUNCATED_PACKAGE_COUNT = 5
+        TRUNCATION_ERROR_MESSAGE = "Results were truncated because too many patches were present, Check log for truncated packages"
```

Comment:
nit: TRUNCATION_WARNING_MESSAGE = "Package lists were truncated to limit reporting data volume. In-VM logs contain complete lists."